src/modules/menu/components/Category.tsx

```diff
@@ -0,0 +1,22 @@
+import RecommendedCategory from "@/modules/menu/components/RecommendedCategory";
+import SimpleCategory from "@/modules/menu/components/SimpleCategory";
+import { Category, RECOMMEND_CATEGORY_ID } from "@/modules/mock/categories";
+import { type Food } from "@/modules/mock/foods";
+
+export type CategoryProps = {
+  category: Category;
+  foods: Food[];
+};
+
+const Category: React.FC<CategoryProps> = ({ category, foods }) => {
+  switch (category?.id) {
+    case RECOMMEND_CATEGORY_ID:
+      return <RecommendedCategory category={category} foods={foods} />;
+    default:
+      return (
+        <SimpleCategory category={category} foods={[...foods, ...foods]} />
```

Comment:
Change 
`<SimpleCategory category={category} foods={[...foods, ...foods]} />`
to
`<SimpleCategory category={category} foods={foods} />`
later